src/sidepanel_unused/pages/LandingPage.module.css

```
@@ -116,9 +122,21 @@
+  width: auto;
+  /* Increased base height for better visibility */
+  height: 32px;
+  max-height: 40px;
   object-fit: contain;
+  /* Center the logo properly */
+  display: block;
+  margin: 0 auto;
+  /* Smooth transitions for hover effects */
+  transition: transform 0.2s ease, opacity 0.2s ease;
+  /* Prevent image dragging */
+  user-drag: none;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)